src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyNotModifiedResponse.java

```diff
@@ -40,7 +41,7 @@
 
     private final int initialWindow;
     private final int requestHash;
-    private final int requestSlot;
+    private int requestSlot;
```

Comment:
If this should be `non-final` then please move it down below all the `final` fields.

Reply:
Done